src/confluence/subcommands/duplicates.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
```

Comment:
I'm not clear on what this subcomment,

```python
confluence duplicates ...
```

is meant to do. Can you please explain in a docstring at the top of this file?

It seems that this is meant to support `confluence list ...`. If that's the case, and this isn't intended to be called as `confluence duplicates ...`, move it to module `base`.